contracts/integrations/test/framework/assertions/finalizePool.ts

```diff
@@ -0,0 +1,269 @@
+import { WETH9Events, WETH9TransferEventArgs } from '@0x/contracts-erc20';
+import { ReferenceFunctions } from '@0x/contracts-exchange-libs';
+import {
+    AggregatedStats,
+    constants as stakingConstants,
+    PoolStats,
+    StakingEpochFinalizedEventArgs,
+    StakingEvents,
+    StakingRewardsPaidEventArgs,
+} from '@0x/contracts-staking';
+import {
+    assertRoughlyEquals,
+    constants,
+    expect,
+    filterLogsToArguments,
+    toDecimal,
+    verifyEventsFromLogs,
+} from '@0x/contracts-test-utils';
+import { BigNumber } from '@0x/utils';
+
+import { DeploymentManager } from '../deployment_manager';
+import { SimulationEnvironment } from '../simulation';
+
+import { FunctionAssertion, FunctionResult } from './function_assertion';
+
+const PRECISION = 15;
+const ALPHA_NUMERATOR = 1;
```

Comment:
If you wanted to use the deployed values they're in [staking/src/constants.ts](https://github.com/0xProject/0x-monorepo/tree/development/contracts/staking/src/constants.ts). I believe the deployed value for alpha is `1/3`.